server/models/historical_validator_stats.ts

```diff
@@ -0,0 +1,62 @@
+import { Validators } from './../../client/scripts/views/pages/validators/index';
+import * as Sequelize from 'sequelize';
+
+export interface HistoricalValidatorStatsAttributes {
+    id: number,
+    stash_id: string;
+    block: string;
+    exposure: string[];
+    commissionPer?: number;
+    preferences?: number;
+    apr?: number;
+    uptime?: string;
+    movingAverages?: number;
+    isLatest: boolean;
+    hasMessage: boolean;
+    isOnline: boolean,
+    eraPoints: number,
+    isElected: boolean,
+    toBeElected: boolean,
+    blockCount: number,
+    otherTotal: string
+}
+
+export interface HistoricalValidatorStatsInstance extends Sequelize.Instance<HistoricalValidatorStatsAttributes>, HistoricalValidatorStatsAttributes {
+
+}
+
+export interface HistoricalValidatorStatsModel extends Sequelize.Model<HistoricalValidatorStatsInstance, HistoricalValidatorStatsAttributes> {
+
+}
+
+export default (
+    sequelize: Sequelize.Sequelize,
+    dataTypes: Sequelize.DataTypes,
+): HistoricalValidatorStatsModel => {
+    const HistoricalValidatorStats = sequelize.define<HistoricalValidatorStatsInstance, HistoricalValidatorStatsAttributes>('HistoricalValidatorStats', {
+        id: { type: dataTypes.INTEGER, autoIncrement: true, primaryKey: true }, //primary-key
+        stash_id: { type: dataTypes.STRING, allowNull: false },
+        block: { type: dataTypes.STRING, allowNull: false }, // blocknumber
+        exposure: { type: dataTypes.JSON, allowNull: false },
+        commissionPer: { type: dataTypes.FLOAT, allowNull: false },
+        preferences: { type: dataTypes.INTEGER, allowNull: false },// preferences - ValidatorPrefs
```

Comment:
validator preferences are just commission so we can remove this: https://github.com/paritytech/substrate/blob/master/frame/staking/src/lib.rs#L447